### ISSUE TITLE
Need help troubleshooting high memory usage issue

### ISSUE THREAD

**Issue by lfcnassif** [libyal/libewf-legacy#17]
Hi @joachimmetz. First, thank you very much for your work with this great library.

Using latest released version 20140812, seems it could be leaking memory reading large E01 images. This was first reported in our project here:
https://github.com/sepinf-inc/IPED/issues/993

Are you aware of some related issue? We would like some advice from you if possible, can [libewf-experimental-20201230](https://github.com/libyal/libewf/releases/tag/20201230) be used for production? Or is libewf-legacy or other stable libewf version still trying to fix memory leaks when confirmed?

Thank you very much in advance.

**Comment by joachimmetz** [comment]
@lfcnassif could you summarize what is the issue you are observing? high memory usage or a leak?

**Comment by lfcnassif** [comment]
Hi @joachimmetz, I posted a summary in the downstream project [here](https://github.com/sepinf-inc/IPED/issues/993#issuecomment-1059777124) as all the details are there. Thanks for your time.

**Comment by joachimmetz** [comment]
Closing in favor of upstream issue
